boards/trellis_m4/src/pins.rs

```diff
@@ -36,6 +44,41 @@ pub struct Accelerometer {
     pub scl: Pa13<Input<Floating>>,
 }
 
+impl Accelerometer {
```

Comment:
Potentially we could add some generics with default values to this type to allow for use with SERCOMs other than 2...

Reply:
Tried that and wasn't happy with the result. SERCOM4 is the only other one compatible with the accelerometer pins, and the original `i2c_master` function is already hardcoded to that...